tests/linearizability/model.go

```diff
@@ -119,6 +129,9 @@ func initState(request etcdRequest, response etcdResponse) EtcdState {
 		} else {
 			state.FailedWrites[request.putData] = struct{}{}
 		}
+	case Delete:
+		//TODO should state have 'deleted' field?
+		state.Value = ""
```

Comment:
```suggestion
		if response.err != nil {
		  state.FailedWrites[""] = struct{}{}
		}
```

Delete can fail, so we should preserve information that.